topi/python/topi/nn/dilation2d.py

```diff
@@ -0,0 +1,151 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+# pylint: disable=invalid-name, unused-variable, too-many-locals
+# pylint: disable=unused-argument, redefined-builtin
+"""Dilation2D operators"""
+from __future__ import absolute_import as _abs
+from tvm import te
+from topi.util import simplify
+from .pad import pad
+from .util import get_pad_tuple
+
+
+def dilation2d_nchw(input, filter, stride, padding, dilation, out_dtype=None):
```

Comment:
In relay op we use ```rate``` as the name for dilation rate, but here we use ```dilation```. Do we want to keep them consistent?

Reply:
Now using 'dilations' instead of 'rates' everywhere. This will be consistent with TF2's Dilation2d op and also with other TVM Conv operators as well.